code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -256,6 +256,8 @@
 	var/static/list/names = list("maintenance pill","floorpill","mystery pill","suspicious pill","strange pill")
 	var/static/list/descs = list("Your feeling is telling you no, but...","Drugs are expensive, you can't afford not to eat any pills that you find."\
 	, "Surely, there's no way this could go bad.")
+	prevent_grinding = TRUE

```

Comment:
You should edit its desc to mention that it isn't dissolvable or grindable.

Reply:
Done!

Reply:
GitHub won't let me edit the desc line above but change it to this:

`desc = "A strange pill found in the depths of maintenance. Somehow, it can't be dissolved or used in a grinder."`